tests/zfs-tests/include/properties.shlib

```diff
@@ -14,7 +14,15 @@
 #
 
 typeset -a compress_prop_vals=('on' 'off' 'lzjb' 'gzip' 'gzip-1' 'gzip-2'
-    'gzip-3' 'gzip-4' 'gzip-5' 'gzip-6' 'gzip-7' 'gzip-8' 'gzip-9' 'zle' 'lz4')
+    'gzip-3' 'gzip-4' 'gzip-5' 'gzip-6' 'gzip-7' 'gzip-8' 'gzip-9' 'zle' 'lz4'
+    'zstd' 'zstd-1' 'zstd-2' 'zstd-3' 'zstd-4' 'zstd-5' 'zstd-6' 'zstd-7'
+    'zstd-8' 'zstd-9' 'zstd-10' 'zstd-11' 'zstd-12' 'zstd-13' 'zstd-14'
+    'zstd-15' 'zstd-16' 'zstd-17' 'zstd-18' 'zstd-19' 
+    'zstd-fast-1' 'zstd-fast-2' 'zstd-fast-3' 'zstd-fast-4'
+    'zstd-fast-5' 'zstd-fast-6' 'zstd-fast-7' 'zstd-fast-8' 'zstd-fast-9'
+    'zstd-fast-10' 'zstd-fast-20' 'zstd-fast-30' 'zstd-fast-40'
+    'zstd-fast-50' 'zstd-fast-60' 'zstd-fast-70' 'zstd-fast-80'
+    'zstd-fast-90' 'zstd-fast-100' 'zstd-fast-500' 'zstd-fast-1000')
```

Comment:
The high amount of zstd algorithms here, decrease the chance of a non-zfs algorithm being chosen significantly.
The same goes for gzip actually.

May I suggest removing some of the gzip (maybe half of them, leave one, remove the other and so forth)
and remove 2/3 of zstd ones.

That would give more balanced results in the tests

Reply:
we dont need balance. all the test are running each compression variant sequencially. there is no random selection for the test i know so far. if there is any, the test has to be modified to use a limited set